### PR TITLE
Release Notes: New $w element - WixComments

### PR DESCRIPTION
**Pull request by wix-docs-feds**

### CODE REVIEW

**Comment by ryangec** [comment]
[WixComments Preview Branch](https://www.wix.com/velo/reference/$w/wixcomments?branch=comments-widget)

File: guides/release-notes.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+### New API: WixComments Element
+
+`Wix Editor Elements ($w)`
+
+Velo APIs exposed the functionality of the [WixComments](https://www.wix.com/velo/reference/$w/wixcomments) element. It is a comment widget available in the [Wix App Market](https://www.wix.com/app-market/comments-widget) to facilitate a comment thread of comments and replies by your site visitors on your Wix site. 
+
+The WixComments API allows you to control the comment widget's functionality using code. You can [lock](https://www.wix.com/velo/reference/$w/wixcomments/lock) the widget, [create threads](https://www.wix.com/velo/reference/$w/wixcomments/setresourceid), [open](https://www.wix.com/velo/reference/$w/wixcomments/opennearestcommentbox) and [close](https://www.wix.com/velo/reference/$w/wixcomments/closecommentboxes) the comment text input area, and set event handlers for [viewing new comments](https://www.wix.com/velo/reference/$w/wixcomments/oncommentcreate) and tracking the total [comment count](https://www.wix.com/velo/reference/$w/wixcomments/oncommentcountchange).
```
Comment (AriGoldWix) [inline guides/release-notes.md:16]: 1. That first mention of WixComments should be in `codeFont`.
2. Does `setResourceId` actually "create threads"?

**Review by AriGoldWix — APPROVED** [review]

File: guides/release-notes.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ We love hearing your feedback! You can help us improve the docs by reporting any
...
+
+`Wix Editor Elements ($w)`
+
+Velo APIs exposed the functionality of the [`WixComments`](https://www.wix.com/velo/reference/$w/wixcomments) element. It is a comment widget available in the [Wix App Market](https://www.wix.com/app-market/comments-widget) to manage comment threads on your Wix site. 
+
+The WixComments API allows you to control the comment widget's functionality using code. You can [lock](https://www.wix.com/velo/reference/$w/wixcomments/lock) the widget, [create new comment threads](https://www.wix.com/velo/reference/$w/wixcomments/setresourceid), [open](https://www.wix.com/velo/reference/$w/wixcomments/opennearestcommentbox) and [close](https://www.wix.com/velo/reference/$w/wixcomments/closecommentboxes) the comment text input area, and set event handlers for [viewing new comments](https://www.wix.com/velo/reference/$w/wixcomments/oncommentcreate) and tracking the total [comment count](https://www.wix.com/velo/reference/$w/wixcomments/oncommentcountchange). 
+
+(May 20, 2024)
+
 ## New API: Dashboard `onBeforeUnload`
```
Comment (AriGoldWix) [inline guides/release-notes.md:21]: Can you add a third # to this one? It looks wrong in the current release notes bc it's the wrong size.

File: guides/release-notes.md [hunk 3]
```diff
@@ -7,6 +7,17 @@ We love hearing your feedback! You can help us improve the docs by reporting any
 ![Was this helpful, yes/no buttons](../media/feedback.png)
 
 ---
+
+### New API: WixComments Element
+
+`Wix Editor Elements ($w)`
+
+Velo APIs exposed the functionality of the [`WixComments`](https://www.wix.com/velo/reference/$w/wixcomments) element. It is a comment widget available in the [Wix App Market](https://www.wix.com/app-market/comments-widget) to manage comment threads on your Wix site.
```
Comment (AriGoldWix) [inline guides/release-notes.md:15]: The new [`WixComments`]() element is a comment widget available...etc.

**Review by AriGoldWix — APPROVED** [review]

**Pull request merged**